packages/expo-auth-session/src/AuthRequest.types.ts

```diff
@@ -82,6 +82,7 @@ export type AuthRequestPromptOptions = Omit<WebBrowserOpenOptions, 'windowFeatur
   /**
    * Should the authentication request use the Expo proxy service `auth.expo.io`.
    * @default false
+   * @deprecated This option will be removed in a future release, check https://expo.fyi/auth-proxy-migration
```

Comment:
```suggestion
   * @deprecated This option will be removed in a future release, for more information check [the migration guide](https://expo.fyi/auth-proxy-migration).
```

You need to use Markdown link to make the URL clickable (this is by design, since we want to avoid raw links), also the added FYI link leads currently to 404 page.

Reply:
Thanks for the suggestion Simek, I've updated this so the URL is clickable. Regarding the link leading to a 404 page Aman will be working on this (https://linear.app/expo/issue/ENG-7628/create-fyi-page-with-instructions-for-how-to-migrate-away-from)